doc/authentication.md

```diff
@@ -60,6 +61,10 @@ cockpit-ws will then send contents of the Authorization http header, without the
 auth scheme, on fd #3 to the command. Once the command has processed the credentials
 it MUST write a JSON response to fd #3.
 
+By default cockpit-ws will wait a maximum of 30 seconds to receive this response.
+This can be adjusted by adding a timeout parameter along side the auth schema
+configuration in your config file as shown above.
```

Comment:
There are also min and max values in the source code. I think they could be mentioned here.